src/stores/AppStore.ts

```diff
@@ -294,6 +295,21 @@ export class AppStore {
         this.initStore();
         break;
       }
+      case AppActionType.SHOW_PREVIEW: {
+        const { view } = action as ShowPreview;
+        const { file } = view;
```

Comment:
can you extract the code from here to the `break` into a private method e.g. `showPreview(view)`?